lms/djangoapps/courseware/views.py

```diff
@@ -855,6 +856,7 @@ def course_about(request, course_id):
     with modulestore().bulk_operations(course_key):
         permission = get_permission_for_course_about()
         course = get_course_with_access(request.user, permission, course_key)
+        modes = CourseMode.modes_for_course_dict(course_key)
 
         if microsite.get_value('ENABLE_MKTG_SITE', settings.FEATURES.get('ENABLE_MKTG_SITE', False)):
```

Comment:
Not really part of this PR but would be nice if we could change this to use theming.helpers.get_value() instead of referencing microsite directly